File: schemacrawler-integrations/src/test/resources/plaintextschema.rb
# encoding: utf-8
puts $catalog.crawlHeaderInfo

for table in $catalog.tables
  # $stderr.puts table.schema
  if table.schema.to_s == "PUBLIC.FOR_LINT"
    next
  end
  # $stderr.puts table.fullName
  puts ''
  puts table.fullName
  for column in table.columns
    puts "  " + column.name
  end
end

File: schemacrawler-tools/src/test/resources/plaintextschema.js
var printCatalog = function()
{
  print(catalog.crawlHeaderInfo);
  
  var schemas = catalog.schemas.toArray();
  for ( var i = 0; i < schemas.length; i++)
  {
    print(schemas[i].fullName);
    var tables = catalog.getTables(schemas[i]).toArray();
    for ( var j = 0; j < tables.length; j++)
    {
      print("o--> " + tables[j].name);
      var columns = tables[j].columns.toArray();
      for ( var k = 0; k < columns.length; k++)
      {
        print("     o--> " + columns[k].name);
      }
    }
  }
};

printCatalog();
